AllSqCalc.java

```diff
@@ -0,0 +1,32 @@
+public class AllSqCalc {
```

Comment:
The name of the class is a bit weird. Would be better to rename it to something meaningful.